sdk/core/azure-servicemanagement-legacy/azure/servicemanagement/_serialization.py

```diff
@@ -31,7 +31,6 @@
     _decode_base64_to_text,
     _encode_base64,
     _lower,
-    _str,
```

Comment:
Please remove all changes of `azure-servicemanagement-legacy`. This package is deprecated, so the code is frozen in time and we won't ever ship it anymore.

Reply:
I have just reset all of the changes in this folder.